### PR TITLE
Qsw 1026 - End to End test improvements

### PR DESCRIPTION
**Pull request by claudio525**

### CODE REVIEW

**Comment by claudio525** [comment]
Do not merge yet, requires another full validation e2e run to ensure everything works.

**Comment by claudio525** [comment]
Ran full validation end to end test without any issues.

File: scripts/cybershake/auto_submit.py [hunk 1]
```diff
@@ -488,13 +516,14 @@ def main(args):
...
                 root_folder,
-                queue_folder,
+                mgmt_queue_folder,
                 binary_mode=binary_mode,
                 hf_seed=hf_seed,
                 rand_reset=rand_reset,
                 extended_period=extended_period,
                 models=(lf_est_model, hf_est_model, bb_est_model, im_est_model),
             )
+            print()
```
Comment (jpa198) [inline scripts/cybershake/auto_submit.py]: Why's this here?

Comment (jasonmotha) [reply scripts/cybershake/auto_submit.py]: Don't forget to flush.

Comment (claudio525) [reply scripts/cybershake/auto_submit.py]: Just a new line for the auto submit output to make it more readable.

**Pull request merged**